openjdk/excludes/ProblemList_openjdk17-openj9.txt

```diff
@@ -254,6 +254,7 @@ java/nio/file/Files/CopyAndMove.java https://github.com/adoptium/aqa-tests/issue
 java/nio/MappedByteBuffer/MapSyncFail.java https://github.com/eclipse-openj9/openj9/issues/6831 generic-all
 jdk/nio/zipfs/ZipFSTester.java https://github.com/eclipse-openj9/openj9/issues/4679 linux-x64,macosx-all
 java/nio/file/Files/probeContentType/Basic.java https://github.com/eclipse-openj9/openj9/issues/14958 windows-all
+java/nio/channels/DatagramChannel/BasicMulticastTests.java https://github.ibm.com/runtimes/backlog/issues/966 macosx-all
```

Comment:
This test should be added around line 235, so tests in the same folder align in this file.